### PR TITLE
Added video walkthrough and some additional resources

### PR DESCRIPTION
**Pull request by davidwrede**
Updated root Getting Started page with video walkthrough and some additional learning resources.

Note: Do not merge this until 4/20 AM.

### CODE REVIEW

File: themes/default/content/docs/get-started/_index.md [hunk 1]
```diff
@@ -43,7 +43,16 @@ Select one of the following options to learn how to deploy cloud resources using
                 </a>
```
Comment (lukehoban) [inline themes/default/content/docs/get-started/_index.md:43]: I think we should also update the intro paragraph to align with updated messaging:

> Pulumi is a modern infrastructure as code platform that allows you to use familiar programming languages and tools to build, deploy, and manage cloud infrastructure.

File: themes/default/content/docs/get-started/_index.md [hunk 2]
```diff
@@ -43,7 +43,16 @@ Select one of the following options to learn how to deploy cloud resources using
...
+        <p class="text-sm leading-relaxed text-gray-600">See <a href="{{< relref "/docs/intro/cloud-providers" >}}">Cloud Providers</a> on how to use other supported clouds.</p>
+    </div>
+</div>
+<div class="md:flex flex-row mt-6 mb-6">
+    <div class="w-full border-solid border-t-2 border-gray-200">
+        <h3 class="no-anchor pt-4"><i class="fas fa-video pr-2"></i>Video Walkthrough</h3>
+        <p>Watch how to deploy a simple application using Pulumi.</p>
+        <div class="md:w-3/4">
+            {{< youtube id="QfJTJs24-JM" title="Introduction to Pulumi: Modern Infrastructure as Code" >}}
+        </div>
```
Comment (lukehoban) [inline themes/default/content/docs/get-started/_index.md:55]: Can we use the same overlay image as on the (new) homepage - and perhaps borrow the styling from there as well - the video looks nicer in that format.

File: themes/default/content/docs/get-started/_index.md [hunk 3]
```diff
@@ -52,3 +61,18 @@ Select one of the following options to learn how to deploy cloud resources using
...
+<div class="md:flex flex-row mt-6 mb-6">
+    <div class="md:w-1/2 border-solid border-t-2 border-gray-200">
+        <h3 class="no-anchor pt-4"><a href="{{< relref "/docs/intro/concepts" >}}"><i class="fas fa-file-alt pr-2"></i>Architecture & Concepts</a></h3>
+        <p>Get details on the Pulumi programming model and core concepts.</p>
+    </div>
+    <div class="md:w-1/2 md:ml-4 border-solid border-t-2 border-gray-200">
+        <h3 class="no-anchor pt-4"><a href="{{< relref "/docs/guides/adopting" >}}"><i class="fas fa-cloud pr-2"></i>Adopting Pulumi</a></h3>
+        <p>Learn how to support, migrate, or convert existing cloud infrastructure with Pulumi.</p>
+    </div>
+</div>
```
Comment (lukehoban) [inline themes/default/content/docs/get-started/_index.md:78]: I do think Tutorials are a particularly valuable thing to link here - as (1) applying Pulumi to a specific problem is one of the most common starting points for users new to Pulumi and (2) they are not particularly prominently linked anywhere else except the end of the Getting Started guides.

File: themes/default/content/docs/get-started/_index.md [hunk 4]
```diff
@@ -52,3 +61,18 @@ Select one of the following options to learn how to deploy cloud resources using
         <p>Deploy your first Pulumi program in a live, <a href="{{< relref "/resources/introduction-to-pulumi" >}}">instructor-led event</a>.</p>
```
[the remark this answers is not in the document]
Comment (davidwrede) [reply themes/default/content/docs/get-started/_index.md:61]: The original concept was to promote live workshops as another way for people to get started with Pulumi (along with the video walkthrough) to accommodate users with different learning styles. I'll put it into the "Additional  resources" section as you suggest to see if that'll accomplish the same goal of increasing traffic to those intro workshops.

File: themes/default/content/docs/get-started/_index.md (hunk 3 above)
Comment (davidwrede) [reply themes/default/content/docs/get-started/_index.md:78]: I agree that they're valuable; however, I don't believe that we should have them be the first place a user goes as part of their initial learning experience. IMHO, they're a little too deep to show someone at the beginning.

We can do a better job of cross-linking, but we have the section link in the left nav and will have one in the top nav under "Learn".

File: themes/default/content/docs/get-started/_index.md (hunk 2 above)
Comment (davidwrede) [reply themes/default/content/docs/get-started/_index.md]: Done. Btw, the image should resolve correctly after everything else is merged.

File: themes/default/content/docs/get-started/_index.md (hunk 1 above)
Comment (davidwrede) [reply themes/default/content/docs/get-started/_index.md]: Fixed in latest commit.

File: themes/default/content/docs/get-started/_index.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (davidwrede) [reply themes/default/content/docs/get-started/_index.md]: Changes in the latest commit.

**Review by lukehoban — APPROVED** [review]
The video is showing a blank image - which I think is expected until other changes are merged - but other than that this looks good!

File: themes/default/content/docs/get-started/_index.md [hunk 5]
```diff
@@ -13,39 +13,62 @@ aliases:
...
+    </div>
+    <div class="md:w-1/2 md:ml-4 border-solid border-t-2 border-gray-200">
+        <h3 class="no-anchor pt-4"><a href="{{< relref "/docs/guides/adopting" >}}"><i class="fas fa-cloud pr-2"></i>Adopting Pulumi</a></h3>
+        <p>Learn how to support, migrate, or convert existing cloud infrastructure with Pulumi.</p>
     </div>
 </div>
+
 <div class="md:flex flex-row mt-6 mb-6">
     <div class="w-full border-solid border-t-2 border-gray-200">
         <h3 class="no-anchor pt-4"><i class="fas fa-users pr-2"></i>Live Workshops</h3>
```
Comment (lukehoban) [inline themes/default/content/docs/get-started/_index.md:51]: This section doesn't seem to have a link on the header, and instead has a hyper-linked word in the description. Should we be consistent between the three sections?

**Comment by davidwrede** [comment]
Yup, I'm expecting the image to resolve after the merge.

File: themes/default/content/docs/get-started/_index.md (hunk 5 above)
Comment (davidwrede) [reply themes/default/content/docs/get-started/_index.md:51]: Good catch. Yup, I'll fix.

**Pull request merged**